### PR TITLE
Fix for existing build failures related to GuideModel classes.

### PR DESCRIPTION
**Pull request by manish2kcoder**
ZypeAppleTvBase project settings updated to include missing file as per conversation with Chris. 

However below changes are not needed in podfile for ZypeAppleTvBase as theese are required for application only -
pod 'ZypeAppleTVBase', :git => 'https://github.com/zype/ZypeAppleTVBase.git'
pod 'Kingfisher', '~> 3.0'
pod 'Analytics', '~> 3.8.0-beta.1'

### CODE REVIEW

File: ZypeAppleTVBase.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -582,21 +586,6 @@
...
-			inputPaths = (
-			);
-			name = "[CP] Copy Pods Resources";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-ZypeAppleTVBase/Pods-ZypeAppleTVBase-resources.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
```
Comment (imchrissmith) [inline ZypeAppleTVBase.xcodeproj/project.pbxproj:599]: @manish2kcoder what does removing this affect?

File: ZypeAppleTVBase.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -612,21 +601,6 @@
...
-			inputPaths = (
-			);
-			name = "📦 Copy Pods Resources";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-ZypeAppleTVBase/Pods-ZypeAppleTVBase-resources.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
```
Comment (imchrissmith) [inline ZypeAppleTVBase.xcodeproj/project.pbxproj:629]: @manish2kcoder what does removing this affect?

File: ZypeAppleTVBase.xcodeproj/project.pbxproj (hunk 1 above)
Comment (manish2kcoder) [reply ZypeAppleTVBase.xcodeproj/project.pbxproj:599]: I intentionally removed this section as we are building our own framework i.e ZypeAppleTvBase and also ZypeAppleTVBase doesn't need any third party dependencies. This Copy pods resources section is applicable to main application which uses ZypeAppleTVBase as cocoa pods dependency. I am not able to think why ZypeAppleTvBase will copy itself to built directory as pods dependency, it should be the work of main apps who will be using ZypeAppleTvBase through pods actually i.e zype-apple-tv in our case?

I am successfully able to build both main application and ZypeAppleTvBase with above changes.

File: ZypeAppleTVBase.xcodeproj/project.pbxproj (hunk 2 above)
Comment (manish2kcoder) [reply ZypeAppleTVBase.xcodeproj/project.pbxproj:629]: same as above.

File: ZypeAppleTVBase.xcodeproj/project.pbxproj (hunk 1 above)
Comment (manish2kcoder) [reply ZypeAppleTVBase.xcodeproj/project.pbxproj:599]: Xcode 11 also flash errors if this section is not removed.

**Pull request merged**